src/api.js

```diff
@@ -29,6 +32,9 @@ export const fetchArticleById = (articleId) => {
   .then(({ data }) => {
     return data.article;
   })
+  .catch((err) => {
+    throw err
```

Comment:
Instead of throwing an error here, you could handle it in the component and display a message to the user if needed.